strategies/failed/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.arquillian.smart.testing</groupId>
+      <artifactId>junit-test-result-parser</artifactId>
+      <version>${project.version}</version>
```

Comment:
Ok then it is another dependency to add when you configure Surefire. Or better in surefire-provider? I agree with you but since now we are only offering one, maybe it is easy in this way of having predefined.